packages/trees/frontend/sirius-components-trees/src/toolbar/TreeToolBar.tsx

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { NewDocumentModal, UploadDocumentModal } from '@eclipse-sirius/sirius-components';
```

Comment:
I've looked into your CI error:

```
@eclipse-sirius/sirius-components-trees:build: [commonjs--resolver] Failed to resolve entry for package "@eclipse-sirius/sirius-components". The package may have incorrect main/module/exports specified in its package.json.
[218](https://github.com/eclipse-sirius/sirius-components/actions/runs/3957509305/jobs/6778760663#step:8:219)
@eclipse-sirius/sirius-components-trees:build: error during build:
[219](https://github.com/eclipse-sirius/sirius-components/actions/runs/3957509305/jobs/6778760663#step:8:220)
@eclipse-sirius/sirius-components-trees:build: Error: Failed to resolve entry for package "@eclipse-sirius/sirius-components". The package may have incorrect main/module/exports specified in its package.json.
```

The issue is coming from the fact that you are depending on `@eclipse-sirius/sirius-components` without specifying it in `package.json` and without modification of the `package-lock.json`. In this specific situation, both modals should only be available in the explorer. You can copy the code of the two modals in the package `sirius-components-tree` and remove them from `sirius-components` entirely. We won't use them outside of the explorer anymore.